0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/utils/create_correlations.py

```diff
@@ -0,0 +1,260 @@
+import pandas as pd
+import numpy as np
+from collections import defaultdict
+import matplotlib.pyplot as plt
+from scipy.stats import gaussian_kde
+
+
+class CreateCorrelations:
+    def __init__(self, platedf, aggregate):
+        """
+        Parameters
```

Comment:
Really good documentation throughout this class for methods and class thank you for making this clean